src/test/java/com/suse/saltstack/netapi/utils/ClientUtilsTest.java

```diff
@@ -3,6 +3,14 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+
```

Comment:
Please remove this line as well, let's put only one empty line to group imports.